File: Telemedicine/NDHM_B2B_B2C/src/app/models/verifyHealthid/PatientDTO.ts
import { AddressDTO } from "./AddressDTO";
import { IdentifierDTO } from "./IdentifierDTO";

export class PatientDTO {
    id: string;

    name: string;

    gender: string;

    yearOfBirth: string;

    address: AddressDTO;

    identifiers: IdentifierDTO[];
}

File: Telemedicine/NDHM/src/main/java/com/nsdl/ndhm/dto/PatientDTO.java
package com.nsdl.ndhm.dto;

import java.util.List;

import lombok.Builder;
import lombok.Data;

@Data
@Builder
public class PatientDTO {

    private String id;

    private String name;

    private String gender;

    private String yearOfBirth;

    private AddressDTO address;
    
    public List<IdentifierDTO> identifiers;

}
